provisioning/roles/mangaki_sync/tasks/main.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: Ensure static files directory to be present
+  file:
+    owner: '{{ mangaki_sync_user }}'
+    group: '{{ mangaki_sync_group }}'
+    mode: 'u=rwx,g=rx,o=rx'
+    path: '{{ mangaki_source_static_root }}'
+    state: 'directory'
+  become: true
+  run_once: true
+  when: 'mangaki_sync_collectstatic|bool'
+  tags: action
```

Comment:
You should also set `recurse: True` in order to create intermediate directories with the same permissions (but I really thought we were already doing it; I'll check).